docs/modifiers/scroll-into-view.md

```diff
@@ -24,6 +24,40 @@ You should use this modifier whenever you need to have an element scrolled into
 
 `shouldScroll` can be either a Boolean or a Promise that resolves to a truthy or falsy value. It does not handle a rejected Promise.
 
+### Usage with focus
+
+When passing in `shouldFocusAfterScroll` as true, it will set focus to the first focusable element found.
+
+```handlebars
+  <div {{scroll-into-view shouldScroll=this.shouldScrollWithFocus options=(hash behavior="smooth") shouldFocusAfterScroll=this.shouldFocusAfterScroll}}>
+    <div>
+      <label for="firstFocusableElement">First Focusable Element: </label>
+      <input name="firstFocusableElement" type="text">
+    </div>
+    <button type="button" {{on "click" this.onScrollIntoViewWithFocus}}>
+      Trigger scroll-into-view and set focus on click
+    </button>
+  </div>
+```
```

Comment:
I almost forgot - can you add a note here about the strange behavior in Safari so developers are aware of it?

You can also add a link to this page as a reference for general browser support (and what happens when `preventScroll` doesn't work), since some consumers may actually care about mobile: https://caniuse.com/mdn-api_htmlelement_focus_options_preventscroll_parameter

Reply:
I didn't mention about Safari because it seems to be working when I tested dummy page on Safari. (It wasn't working in my app when I tried this change probably because something to do with the app)

A note for browser compatibility of `preventScroll` is a great idea, lemme add that. Thanks!